packages/api/src/external/commonwell/patient.ts

```diff
@@ -945,14 +945,8 @@ async function updatePersonAndLink({
     throw err;
   }
 
-  const { networkLinks } = await findOrCreatePersonAndLink({
-    commonWell,
-    queryMeta,
-    commonwellPatient,
-    commonwellPatientId,
-    patientRefLink,
-    patient,
-  });
+  const networkLinks = await getPatientsNetworkLinks(commonWell, queryMeta, commonwellPatientId);
```

Comment:
In [this PR ](https://github.com/metriport/metriport/pull/3288/files#diff-99dc50a6fe28511e6bd6fadca6d31fd7ca4624d4258bef3e291112c0a7c55cca) we used to use autoUpgradeNetworkLinks which doesnt try to findOrCreatePerson and add patient link like `findOrCreatePersonAndLink` does. So just updating this to simply get the links.